packages/headless/src/controllers/smart-snippet-questions-list/headless-smart-snippet-questions-list.ts

```diff
@@ -98,6 +102,37 @@ export interface SmartSnippetQuestionsList extends Controller {
    * @param identifier - The `questionAnswerId` of the smart snippet to collapse.
    */
   cancelPendingSelectSource(identifier: string): void;
+  /**
+   * Selects a link inside an answer, logging a UA event to the Coveo Platform if it was never selected before.
+   *
+   * In a DOM context, we recommend calling this method on all of the following events:
+   * * `contextmenu`
+   * * `click`
+   * * `mouseup`
+   * * `mousedown`
```

Comment:
Just to make sure: currently it seems like people should call this method both when someone presses down on their mouse, and when they release it. Is that correct?

Reply:
Yes, that's correct.